chrome-launcher/chrome-launcher.ts

```diff
@@ -45,24 +48,79 @@ export interface ModuleOverrides {
   spawn?: typeof childProcess.spawn;
 }
 
-export async function launch(opts: Options = {}): Promise<LaunchedChrome> {
-  opts.handleSIGINT = defaults(opts.handleSIGINT, true);
+/**
+ * LaunchManager handles launch retries and SIGINT binding
+ */
+export class LaunchManager {
+  private instance?: Launcher;
+  private opts: Options;
```

Comment:
oh I'm just confused how typescript can let this compile when it's not initialized in the constructor, it will definitely be undefined...